hazelcast/src/test/resources/hazelcast-fullconfig.xml

```diff
@@ -273,6 +274,10 @@
 
         <wan-replication-ref name="my-wan-cluster">
             <merge-policy>hz.PASS_THROUGH</merge-policy>
+            <filters>
+                <filter>MYFILTER_1</filter>
```

Comment:
We don't have any out of the box filter implementation. What I can do is, changing this to `com.example.SampleFilter` to emphasize that this field needs a fully qualified class name.